js/assessment.js

```diff
@@ -242,7 +242,7 @@ define([
 
         get: function(id) {
             if (id === undefined) {
-                return this._assessments.slice(0);
+                return this._assessments[0];
```

Comment:
I still don't think the approach you've taken solves the problem you've described.

https://github.com/adaptlearning/adapt-contrib-assessmentResults/blob/master/js/adapt-contrib-assessmentResults.js#L225
```javascript
var assessmentModel;
if (!state.id) {
assessmentModel = Adapt.assessment.get()[0];
} else {
assessmentModel = Adapt.assessment.get(state.id);
}
```
^ if no id defined in assessmentResults, pick the first one.

https://github.com/adaptlearning/adapt-contrib-assessment/blob/master/js/adapt-assessmentArticleModel.js#L108
```javascript
var assessmentConfig = this.get("_assessment");
if (!assessmentConfig._id) {
    assessmentConfig._id = this.model.get("_id");
}
```
^ if no id defined in assessment, take article id